pytorch_blade/mnist.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2022 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
If you want to move some files, please use `git mv` instead of the Linux copy command to keep git history, which helps to revert commits if necessary in the future.